logging/bin/getlogs.py

```diff
@@ -8,132 +8,66 @@
 from opensearchpy import OpenSearch
 import os, sys
 import json, csv
+import tempfile
 
-## Cleanup temp file if not previously deleted, Create target directory
-DEFAULT_DIR = "GetLogsFiles"
-if not os.path.isdir(DEFAULT_DIR):
-    os.mkdir(DEFAULT_DIR)
-
-if os.path.isfile(DEFAULT_DIR + "/temp_SAS_OSQ.TXT"):
-    os.remove(DEFAULT_DIR + "/temp_SAS_OSQ.TXT")
+##v 0.2.0
 
 def validate_input(dict):
-    """Validate the arguments passed by the user to ensure script will function properly"""
-    
-    if(len(sys.argv) == 1): ##Check if any args have been provided
-        print("No arguments have been provided")
-        exit()
-
-    if dict['create-auth']:
-        if (type(dict['create-auth'] == list)):
-            dict['create-auth'] = " ".join(dict['create-auth'])
-        if (not (DEFAULT_DIR + "/") in  dict['create-auth']):
-            dict['create-auth'] = os.path.join(DEFAULT_DIR, dict['create-auth'])
-        x = open(dict['create-auth'], "w")
-        x.write('{"username": " " , "password": " " , "host": " ", "port": " "}')
-        print("\nAuth File created.\n")
-
-    if dict['auth-file']:
-        if (type(dict['auth-file'] == list)):
-            dict['auth-file'] = " ".join(dict['auth-file'])
-        if (not (DEFAULT_DIR + "/") in  dict['auth-file']):
-            dict['auth-file'] = os.path.join(DEFAULT_DIR,  dict['auth-file'])
-        try: 
-            a = open(dict['auth-file'], 'r')
-            authDict = json.loads(a.read())
-            dict['userName'] = authDict['username']
-            dict['password'] = authDict['password']
-            dict['host'] = authDict['host']
-            dict['port'] = authDict['port']
-        except Exception as e:
-            print("Error: There was a problem with the authentication file. Please ensure that the values are in dictionary format with keys: 'username', 'password', 'host', 'port'")
-            print(e)
-            exit()
-
-    if dict['format']:
-        if (dict['format'].lower() != 'csv') and (dict['format'].lower() != 'json'):
-            print("Error: Not a valid output type for --format. Supported filetypes are csv and json.")
-            exit()
-        else:
-            dict['format'] = dict['format'].lower()
+    ## Validate the arguments passed by the user to ensure script will function properly
```

Comment:
This should stay with 3 quotes as it is Python standard for doc string to explain purpose of a function, this is not a normal comment.